system-addon/test/unit/common/ShortUrl.test.js

```diff
@@ -1,6 +1,21 @@
-const {shortURL} = require("common/ShortURL.jsm");
+const {shortURL} = require("lib/ShortURL.jsm");
```

Comment:
This test should also be moved. Probably `system-addon/test/unit/lib/ShortUrl.test.js`